lib/pharos/phases/label_node.rb

```diff
@@ -15,30 +15,26 @@ def call
         raise Pharos::Error, "Cannot set labels, node not found" if node.nil?
 
         logger.info { "Configuring node labels and taints ... " }
-        patch_labels(node) if @host.labels
+        patch_labels(node) unless @host.labels.empty?
         patch_taints(node) if @host.taints
       end
 
       # @param node [K8s::Resource]
       def patch_labels(node)
-        kube_nodes.update_resource(
-          node.merge(
-            metadata: {
-              labels: @host.labels
-            }
-          )
-        )
+        kube_nodes.merge_patch(node.metadata.name, {
```

Comment:
Not sure why I did this... maybe I should revert these back to `update_resource`?

Reply:
👍